olp-cpp-sdk-core/src/http/winhttp/NetworkWinHttp.cpp

```diff
@@ -174,8 +174,12 @@ std::wstring ProxyString(const olp::http::NetworkProxySettings& proxy) {
 
   switch (proxy.GetType()) {
     case olp::http::NetworkProxySettings::Type::NONE:
+    case olp::http::NetworkProxySettings::Type::HTTP:
       proxy_string_stream << "http://";
       break;
+    case olp::http::NetworkProxySettings::Type::HTTPS:
+      proxy_string_stream << "https://";
```

Comment:
So it is enough to set "https://" and it works?

Reply:
No, it's not. I already added log message that it's not supported in Windows.